### ISSUE TITLE
URL.createObjectURL fails on Safari 9.0.3 and iOS-devices 

### ISSUE THREAD

**Issue by bildsoe** [localForage/localForage#679]
Hi, thanks for a super project.

I'm trying to store image data as a blob and then read it later and create an image from it using URL.createObjectURL. When I look at the databases in the Storage tab in Safari 9.0.3 I see that each of my blobs starts with __lfsc__:blob~~local_forage_type~~ - Can this be the cause of my problems? Or is this removed when using localforage.getItem?

I create the objectURL using this line:
myimage.src = URL.createObjectURL(new Blob([data],{type: 'image/png'}));

Its works on Chrome and Android devices.

BR
Thomas

**Comment by thgreasi** [comment]
It deseriaize properly on its own. Did you face any issue while developing?

**Comment by bildsoe** [comment]
No, no problems as I developed it. I was testing mainly on desktop Chrome.

But Perhaps I should try setting the Blob type before setItem?

**Comment by bildsoe** [comment]
I'm currently testing on iOS 10.2 - and here I have the problem with the deserialized data.

**Comment by bildsoe** [comment]
So I'm finished testing and Blob-type is important :) - It should be set on both setItem and getItem otherwise the blobdata won't work on iOS.
